elasticsearch/templates/statefulset.yaml

```diff
@@ -174,8 +174,13 @@ spec:
             value: "{{ .Values.minimumMasterNodes }}"
           {{- end }}
           {{- end }}
+          {{- if le (int .Values.esMajorVersion) 7 }}
           - name: discovery.zen.ping.unicast.hosts
             value: "{{ template "masterService" . }}-headless"
+          {{- else }}
```

Comment:
Could you add a test for this? You can more or less copy paste this existing one https://github.com/elastic/helm-charts/blob/c5289d5c3e78820d78baf0984c439c8ed68c893f/elasticsearch/tests/elasticsearch_test.py#L230-L246

Reply:
sure